tests/lib/collectors/grr_base.py

```diff
@@ -62,7 +65,8 @@ def testApprovalWrapper(self):
     grr_base_module.setup('random reason',
                           'http://fake/endpoint',
                           ('admin', 'admin'),
-                          'approver1@google.com,approver2@google.com')
+                          'approver1@google.com,approver2@google.com',
```

Comment:
Not important, but would change these to approver1@example.com ..

Reply:
done.